### DISCUSSION TITLE
Weather Widget in task bar (like

### DISCUSSION THREAD

**Discussion by MrSmile83** [valinet/ExplorerPatcher#616] [Ideas]
Is it posible to get the new weather widget in the left side like the last Windows 11 inside version in the future compilations? Thanks in advance.

![image](https://user-images.githubusercontent.com/11159601/147859371-928a433c-d0c9-418d-8cee-a51c3dab763e.png)

**Comment by valinet** [comment]
Chances are very slim to none. The easiest is for Microsoft to backport that feature. More chances being able to introduce some weather widget for the Windows 10 taskbar, although, yeah, someone’s gotta work on that as well. The explorer.exe shipped with build 22000 doesn’t have the news and interests feature at all, it’s simply not there, not inactive like many other things.

**Comment by MrSmile83** [comment]
Thanks for your answer. I supose that the windows 10 news and interests features not is posible to make to work it in Windows 11 taskbar, is it true?

**Reply by valinet** [reply]
Yup, think so. Unlikely. Although everything's possible. Anyway, if I were to work on anything, it would definitely be for the old Windows 10 taskbar (which, I mean, you can kind of tell by looking at ExplorerPatcher). The new one is hostile to third party development, as is most of Microsoft's stuff, attitude and decisions lately, so I wouldn't want to contribute to it anyway.

**Comment by Anixx** [comment]
No, of course. Those who want weather are considered perverts and reported to the police

**Reply by valinet** [reply]
Huh?
